_rules/object-has-acessible-name-8fc3b6.md

```diff
@@ -18,7 +23,7 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any `object` element that is [included in the accessibility tree][].
+This rule applies to any `object` element that is [included in the accessibility tree][] and embeds a resource with an [image MIME type](https://mimesniff.spec.whatwg.org/#image-mime-type) or an [audio or video MIME type](https://mimesniff.spec.whatwg.org/#audio-or-video-mime-type).
```

Comment:
I like that ❤️ It does raise a few questions…

1. Should we have examples with a `type` attribute?
2. How is the type determined without `type` attribute? Looks like it is step 8 in the algo at https://html.spec.whatwg.org/multipage/iframe-embed-object.html#the-object-element which is somewhat complex :-/
The question is especially relevant in "are we sure that the examples are going to be served with the correct MIME type?"
3. Can we add more types to this list (e.g. archives, fonts, …)? This is definitely not needed for the validity of the rule…
I think it might be worth to add a definition of "non-text MIME type" which would them be easy to update if needed (plus, can be reused, and makes the intention super clear in that rule: "an `object` which embeds a [non-text][non-text mime] resource")

Reply:
@Jym77 I'll start by addressing question 2 and then 1 (before question 3)

2. I don't think we should dive into the algorithm (similar to what we do with accessible names, for instance). The browser determines the mime type of the resource (in the same way it builds the accessibility tree). I admit I haven't looked into the details of the process, but by looking at the headers in the HTTP response to the request for the object, the mime type is there and it is the correct one for all the examples we have. 
Furthermore, even when I enter an incorrect mime type using the type attribute, I made two observations: 1) the response headers always have the correct type (ignoring what I entered in the type attribute); 2) the browser always renders the object correctly, apparently also ignoring the type attribute.

1. Given the observations above, I don't really think we need to have examples with the type attribute, because it doesn't seem to be used for any rendering decisions.

3. Not sure it that would be useful. SC 1.1.1 addresses non-text content that is presented to the user. Fonts and archives are not really presented to user, are they? And if they are, I supposed they would be presented as text. By including those, we then would have to write the applicability in a way to leave them out.

I'll wait for your input, to understand if you think we still need to change anything in the rule!

Reply:
I'm OK with that 👍